gruntfile.js

```diff
@@ -11,10 +11,18 @@ module.exports = function(grunt) {
         }
       }
     },
-    concat: {
+  concat: {
       dist: {
-        src: ['assets/js/app.js', 'assets/js/api.js'],
-        dest: 'assets/js/production.js'
+        src: [
+          'assets/js/app.js',
+          'assets/js/ui.js',
+          'assets/js/api.js'
+        ],
+        dest: 'assets/js/production.js',
```

Comment:
Next round of refactoring, we should have the build process create a 'build' or 'dist' directory that is separate from the source code. It is confusing to have the generated file 'production.js' in with the source code. Similar comment for the 3rd party libraries, which should be in a 'libs' or 'vendor' directory, rather than the urbandev source directory.